modules/extensions-support/src/main/java/org/mule/module/extension/internal/manager/DefaultExtensionManager.java

```diff
@@ -50,6 +58,7 @@
     private ObjectNameHelper objectNameHelper;
     private ExtensionDiscoverer extensionDiscoverer = new DefaultExtensionDiscoverer(new DefaultExtensionFactory(serviceRegistry), serviceRegistry);
     private ImplicitConfigurationFactory implicitConfigurationFactory;
+    private ConfigurationExpirationMonitor configurationExpirationMonitor;
```

Comment:
Why there is only a ConfigurationExpirationMonitor for all the different types of available extensions?